README.md

```diff
@@ -41,6 +41,7 @@ module "postgresql" {
 |------|---------|
 | <a name="provider_azuread"></a> [azuread](#provider\_azuread) | n/a |
 | <a name="provider_azurerm"></a> [azurerm](#provider\_azurerm) | >= 3.7.0 |
+| <a name="provider_azurerm.aks_subscription"></a> [azurerm.aks\_subscription](#provider\_azurerm.aks\_subscription) | >= 3.7.0 |
```

Comment:
maybe `postgres_network`?

it's nothing to do with aks it's the provider that has the subscription that the subnet is in.

Reply:
yeah, I dont mind - was just copying similar to Jenkins, updated.